execution_engine/src/core/runtime/mod.rs

```diff
@@ -2592,17 +2558,14 @@ where
             Groups::default(),
         );
 
-        let value = StoredValue::ContractPackage(contract_package);
-
-        Ok((value, access_key))
+        Ok((contract_package, access_key))
     }
 
     fn create_contract_package_at_hash(&mut self) -> Result<([u8; 32], [u8; 32]), Error> {
```

Comment:
Just noticing this type signature for the first time: really unfortunate that we have a tuple of naked byte arrays.  Type aliases or even just a doc-string could go a long way in helping understanding **what** the return type is.

Reply:
Agreed. I ticketed it here: https://casperlabs.atlassian.net/browse/EE-1096. It's a relatively straightforward refactor